spec/unit/serializers/team_list_serializer_spec.rb

```diff
@@ -24,8 +24,9 @@
 
   context 'Some files transferred' do
```

Comment:
same as with the other test class.

I know, this is not really related to the recrypt feature. but would be nice to clean it up anyway